File: easywk-parser-dotnet/StarterLine.cs
namespace easywk_parser_dotnet;

public class StarterLine
{
    public readonly string Wettkampf;
    public readonly string Lauf;
    public readonly string Bahn;
    public readonly string Teilnehmer;
    public readonly string Verein;
    public readonly string Meldezeit;

    public StarterLine(string wettkampf, string lauf, string bahn, string teilnehmer, string verein, string meldezeit)
    {
        Wettkampf = wettkampf;
        Lauf = lauf;
        Bahn = bahn;
        Teilnehmer = teilnehmer;
        Verein = verein;
        Meldezeit = meldezeit;
    }

    public override string ToString()
    {
        return $"{Wettkampf}\t{Lauf}\t{Bahn}\t{Teilnehmer}\t{Verein}\t{Meldezeit}";
    }
}

File: easywk-parser-node-ts/src/StarterLine.ts
export class StarterLine {
    public readonly wettkampf: string;
    public readonly lauf: string;
    public readonly bahn: string;
    public readonly teilnehmer: string;
    public readonly verein: string;
    public readonly meldezeit: string;


    constructor(wettkampf: string, lauf:string, bahn:string, teilnehmer: string, verein: string, meldezeit: string) {
        this.wettkampf = wettkampf;
        this.lauf = lauf;
        this.bahn = bahn;
        this.teilnehmer = teilnehmer;
        this.verein = verein;
        this.meldezeit = meldezeit;
    }

    public toString(): string {
        return this.wettkampf+"\t"+this.lauf+"\t"+this.bahn+"\t"+this.teilnehmer+"\t"+this.verein+"\t"+this.meldezeit;
    }
}